.pipelines/android_packaging.yml

```diff
@@ -0,0 +1,78 @@
+# packaging pipeline for onnxruntime-extensions Android AAR package
+
+parameters:
+  - name: IsReleaseBuild
+    displayName: "Is this is a release build?"
+    type: boolean
+    default: false
+
+name: "$(Date:yyyyMMdd)$(Rev:rrr)" # build number format
+
+jobs:
+  - job: AndroidPackaging
+    pool:
+      vmImage: "macOS-12"
+    timeoutInMinutes: 120
+    variables:
+      OnnxruntimeDirectory: cmake/external/onnxruntime
+    steps:
+      - script: brew install coreutils ninja
+        displayName: Install coreutils and ninja
+
+      - script: |
+          mkdir cmake/external
+          cd cmake/external
+          git clone https://github.com/microsoft/onnxruntime.git
+          cd onnxruntime
+          git checkout v1.12.1
+        displayName: Download Onnxruntime
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          python3 tools/python/run_android_emulator.py \
+            --android-sdk-root ${ANDROID_SDK_ROOT} \
+            --create-avd --system-image "system-images;android-30;google_apis;x86_64" \
+            --start --emulator-extra-args="-partition-size 4096" \
+            --emulator-pid-file $(Build.BinariesDirectory)/emulator.pid
+        displayName: Start Android emulator
+        workingDirectory: $(OnnxruntimeDirectory)
+
+      - task: JavaToolInstaller@0
+        displayName: Use jdk 11
+        inputs:
+          versionSpec: "11"
+          jdkArchitectureOption: "x64"
+          jdkSourceOption: "PreInstalled"
+
+      # TODO: set package version number here
```

Comment:
currently we need to do a build first so CMake generates `.version.txt`. some ideas to handle that:
- check in version.txt in the repo and derive the cmake project version variables from it
- allow a version suffix to be specified for a package